python/mxnet/test_utils.py

```diff
@@ -1521,6 +1522,72 @@ def download(url, fname=None, dirname=None, overwrite=False, retries=5):
     logging.info("downloaded %s into %s successfully", url, fname)
     return fname
 
+def download_model(model_name, dst_dir='./', meta_info=None):
```

Comment:
should this be a separate PR?

Reply:
no this is needed for this PR so I added as part of this PR. I thought this was small enough to be included in this PR.